Makefile

```diff
@@ -390,13 +402,17 @@ CI_JS_SUITES ?= default
 CI_DOC := doctool
 
 # Build and test addons without building anything else
+# The CI runs this on ARM(Raspberry Pis) because the addons and
```

Comment:
Uh, wait, I don't think this is true..? From what I can tell `node-test-binary-arm` only runs `test-ci-js`, But then I am not sure who uses this target and how are the addons tested on the pis @nodejs/build

Reply:
node-test-binary-arm:

```
case $RUN_SUBSET in
  addons)
    PYTHON=python FLAKY_TESTS=$FLAKY_TESTS_MODE make test-ci-native
    ;;
  *)
    export CC=should-not-compile CXX=should-not-compile # Prevent building
    PYTHON=python FLAKY_TESTS=$FLAKY_TESTS_MODE TEST_CI_ARGS="--run=${RUN_SUBSET},7" make test-ci-js
    ;;
esac
```

i.e. it's used for the addons tests on arm-fanned.